gr-dtv/lib/atsc/interleaver_fifo.h

```diff
@@ -22,52 +23,40 @@ class interleaver_fifo
 {
 public:
     interleaver_fifo(unsigned int size);
```

Comment:
What is the difference between this FIFO and the one in `atsc_interleaver_fifo.h`? Can they be merged into one?

Reply:
I don't know. I'd say that would make for a separate PR, though.